include/unifex/detail/intrusive_heap.hpp

```diff
@@ -32,7 +32,7 @@ class intrusive_heap {
       assert(item->*Prev == nullptr);
     }
     while (item != nullptr) {
-      std::printf("- %p\n", item);
+      printf("- %p\n", item);
```

Comment:
I think this was missing `#include <cstdio>` rather than being incorrectly scoped.
Although I'd like to replace this with some kind of `TRACE()` macro at some point that allows it to be compiled out if required.